Changelog.md

```diff
@@ -14,13 +14,15 @@
 
 - [#609]: Added `Writer::write_serializable` to provide the capability to serialize
   arbitrary types using serde when using the lower-level `Writer` API.
+- [#615]: Added ability to set entity resolver when deserialize using borrowing reader.
 
 ### Bug Fixes
 
 ### Misc Changes
 
 
 [#609]: https://github.com/tafia/quick-xml/issues/609
```

Comment:
This should be pull #609, not issue #609

My mistake, looks like I didn't fix it up completely.

Reply:
Ok, will fix. Just a moment...